DuplicateFinderEngine/ScanEngine.cs

```diff
@@ -64,12 +63,23 @@ public class ScanEngine {
 			DatabaseHelper.SaveDatabase(DatabaseFileList);
 		}
 
+		public async void PopulateDuplicateThumbnails() {
+			await Task.Run(() => PopulateThumbnails(m_cancelationTokenSource.Token));
+			ThumbnailsPopulated(this, null);
+		}
+
 		public async void CleanupDatabase() {
 			await Task.Run(() => DatabaseHelper.CleanupDatabase(DatabaseFileList));
 			DatabaseCleaned?.Invoke(this, null);
 		}
-		public async void ExportDatabaseVideosToCSV() {
-			await Task.Run(() => DatabaseHelper.ExportDatabaseVideosToCSV(DatabaseFileList));
+		public async void ExportDatabaseVideosToCSV(bool onlyVideos, bool onlyFlagged) {
```

Comment:
Wouldn't it be better using the flag as paramater you created in VideoFileEntry.cs and then just provide the flags to include / exclude?

Reply:
i figured there are many ways we might want to use the flags the in the future, right now the expression is: 
(f == ManuallyExcluded || f== AllErrors) && f == IsImage
chances are, if the next version needs more parantheses, a generic solution might need to be changed again